src/pages/loading-waiting-states/skeletons/Skeletons.tsx

```diff
@@ -0,0 +1,359 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles, useTheme, Theme } from '@material-ui/core/styles';
+import {
+    AppBar,
+    Toolbar,
+    Hidden,
+    IconButton,
+    Typography,
+    Button,
+    ButtonGroup,
+    Tooltip,
+    Card,
+    List,
+} from '@material-ui/core';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { useDispatch } from 'react-redux';
+import { Cloud, KeyboardArrowRight, ListAlt, Menu, Notifications, Refresh } from '@material-ui/icons';
+import { ScorecardPlaceholder } from './components/ScorecardPlaceholder';
+import { ListItemDensePlaceholder, ListItemPlaceholder } from './components/ListItemPlaceholder';
+import { HeroBannerPlaceholder } from './components/HeroBannerPlaceholder';
+import { Hero, HeroBanner, InfoListItem, ScoreCard, Spacer } from '@pxblue/react-components';
+import * as colors from '@pxblue/colors';
+import { CurrentCircled, GradeA, Temp, Device, Moisture } from '@pxblue/icons-mui';
+
+const backgroundImage = require('../../../assets/topology_40.png').default;
+
+const useStyles = makeStyles((theme: Theme) => ({
+    root: {
+        backgroundColor: theme.palette.background.paper,
+        minHeight: '100vh',
+    },
+    appbarRoot: {
+        padding: 0,
+    },
+    toolbarGutters: {
+        padding: `0 ${theme.spacing(2)}px`,
+    },
+    exampleContainer: {
+        margin: `${theme.spacing(3)}px ${theme.spacing(2)}px`,
+    },
+    buttonRow: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        marginBottom: theme.spacing(2),
+    },
+    label: {
+        marginRight: theme.spacing(2),
+    },
+    selected: {
+        backgroundColor: theme.palette.primary.light,
+        color: theme.palette.primary.main,
+    },
+    title: {
+        marginTop: theme.spacing(5),
+        marginBottom: theme.spacing(3),
+        color: theme.palette.primary.main,
+    },
+    heroBannerCard: {
+        width: 384,
+        height: 132,
+    },
+    leftComponent: {},
+    abbreviation: {
+        fontWeight: 600,
+        marginLeft: 2,
+    },
+}));
+
+export const Skeletons = (): JSX.Element => {
+    const theme = useTheme();
+    const classes = useStyles(theme);
+    const dispatch = useDispatch();
+    const [isLoading, setIsLoading] = useState(true);
+    const [animationStyle, setAnimationStyle] = useState<'pulse' | 'wave'>('pulse');
+    let refreshTimeout: ReturnType<typeof setTimeout>;
+
+    const refreshData = (): void => {
+        setIsLoading(true);
+
+        refreshTimeout = setTimeout((): void => {
+            setIsLoading(false);
+        }, 3000);
+    };
```

Comment:
let's match angular and just do this on a continuous loop instead of a button press.